### PR TITLE
[FINAL] Fix warnings

### PR DESCRIPTION
**Pull request by tonidero**
Just fixed some warnings on unused code

### CODE REVIEW

File: src/moaiext-iphone/MOAIGameCenterIOS.h [hunk 1]
```diff
@@ -1,92 +0,0 @@
-// Copyright (c) 2010-2011 Zipline Games, Inc. All Rights Reserved.
-// http://getmoai.com
-
-#ifndef	MOAIGAMECENTERIOS_H
-#define	MOAIGAMECENTERIOS_H
```
Comment (aboedo) [inline src/moaiext-iphone/MOAIGameCenterIOS.h:5]: how will the 3 users of iOS game center get their stats integrated?

**Review by aboedo — COMMENTED** [review]
LGTM, looks like it's mostly stuff we don't use and don't intend to use ever.

**Review by aboedo — APPROVED** [review]

File: src/moaiext-iphone/MOAIGameCenterIOS.h (hunk 1 above)
Comment (tonidero) [reply src/moaiext-iphone/MOAIGameCenterIOS.h:5]: "Sorry but that's not a feature we currently support. We might consider adding it in the future" (Probably never :P )

**Review by stevenmah — APPROVED** [review]
:shipit:

**Pull request merged**